util/Migrator/DbScripts/2024-10-16_00__AddIsMultiOrgEnterpriseColumn.sql

```diff
@@ -0,0 +1,175 @@
+-- Table
+IF OBJECT_ID('[dbo].[ClientOrganizationMigrationRecord]') IS NULL
```

Comment:
❌ It looks like you might have forgot to update the SQL Server migration script. This is just creating an unrelated table that already exists. Yours will look more like [this](https://github.com/bitwarden/server/blob/9f7e05869e2a0040b5b5e6197398f82c71e75b13/util/Migrator/DbScripts/2024-03-07_00_SetupProviderBilling.sql), but just with the new `IsMultiOrgEnterprise` column.